### PR TITLE
ios: handle av system pressure level updates

### PR DESCRIPTION
**Pull request by alexbumbu**
Handle AV system pressure level updates received when using the app in multitasking on iPad devices

### CODE REVIEW

File: ios/RCTWebRTC/VideoCaptureController.m [hunk 1]
```diff
@@ -45,88 +48,123 @@ -(instancetype)initWithCapturer:(RTCCameraVideoCapturer *)capturer
...
+        = [self selectFormatForDevice:self.device
+                      withTargetWidth:self.width
+                     withTargetHeight:self.height];
+    if (!format) {
+        RCTLogWarn(@"[VideoCaptureController] No valid formats for device %@", self.device);
 
         return;
     }
 
-    _selectedFormat = format;
```
Comment (alexbumbu) [inline ios/RCTWebRTC/VideoCaptureController.m:84]: will fix this

File: ios/RCTWebRTC/VideoCaptureController.m [hunk 2]
```diff
@@ -165,4 +203,19 @@ - (AVCaptureDeviceFormat *)selectFormatForDevice:(AVCaptureDevice *)device
...
+- (void)throttleFrameRateForDevice:(AVCaptureDevice *)device {
+    NSError *error = nil;
+    
+    [device lockForConfiguration:&error];
+    if (error) {
+        RCTLog(@"[VideoCaptureController] Could not lock device for configuration: %@", error);
+        return;
+    }
+    
+    device.activeVideoMinFrameDuration = CMTimeMake(1, 20);
```
Comment (saghul) [inline ios/RCTWebRTC/VideoCaptureController.m:215]: Are these in the right order? We are setting this to 15fps min and 20 fps max, right?

Comment (saghul) [reply ios/RCTWebRTC/VideoCaptureController.m:215]: Never mind: "A device’s minimum frame duration is the reciprocal of its maximum frame rate. "

Comment (alexbumbu) [reply ios/RCTWebRTC/VideoCaptureController.m:218]: yeah, found this a bit confusing too

**Review by saghul — APPROVED** [review]
LGTM excellent work!

**Pull request merged**